schema/release-schema.json

```
@@ -525,7 +525,7 @@
           }
         },
         "amendments": {
-          "description": "A tender amendment is a formal change to the tender, and generally involves the publication of a new tender notice/release. The rationale and a description of the changes made can be provided here.",
+          "description": "Formal changes to the tender, which generally involve the publication of a new notice/release.",
```

```suggestion
          "description": "The formal changes to the tender, which generally involve the publication of a new notice/release.",
```

more in keeping with the style guide to start with an [article](https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#articles)